src/main/webapp/i18n/en/plagiarism.json

```diff
@@ -116,7 +116,8 @@
                 "exportCsv": "Export CSV",
                 "confirm": "Confirm case",
                 "discard": "Discard case"
-            }
+            },
+            "continuousPlagiarismControlTooltip": "Continuous Plagiarism Control triggers plagiarism checks automatically each night."
```

Comment:
"checks" or check". It's singular in German, plural in English.

Reply:
Changed to _check_